keps/sig-scheduling/5075-dra-consumable-capacity/README.md

```diff
@@ -0,0 +1,1519 @@
+- Options must be a list of unique values.
+- The minimum must be less than or equal to the maximum in the `valueRange`.
+- If a chunk size is defined, both the default and the maximum must be multiples of the chunk size.
+- The minimum, maximum, and (minimum + chunk size) must each be less than the capacity value.
+
+**Distinct Attribute**
+
+- Similar to the `matchAttribute`, check for a missing domain and required name (invalid request).
+- If the feature gate is enabled, exactly one of `matchAttribute` or `distinctAttribute` must be provided.
+- If the feature gate is disabled, `matchAttribute` is required.
+
+**Device Name and Share ID**
+
+- If the feature gate is enabled and multi-allocation is allowed, ensure the device domain name length is sufficient: it must be at least (share ID size + 1).
+- The share ID must have a designated size and must be decodable as hex.
+- When this feature gate and `deviceStatusFeatureGate` ([KEP 4817](https://github.com/kubernetes/enhancements/issues/4817)) are enabled, the `device` field in the allocated device status may contain a `/`. In that case, the first part must be validated as a device name, and the second part as a share ID. This extended device name should enable a one-to-one mapping with the allocation result which contains a share ID.
+- If the feature gate is disabled, share ID and allow multi-allocation should be ignored. The device name with `/` should be failed.
+
+###### Allocator
+
+**Allow Multiple Allocations**
+
+- can allocate a device which allow multiple allocations for multiple times
+- must not allocate a device which do not allow multiple allocations more than once
+- can exclude dedicated device from allocation with CEL
+- can limit allocation to multi-allocatable device with CEL
+- can work with partitionable devices
+
+**Consumable Capacity**
+
+- can gather consumed capacity from allocated resource claims
+- can add/remove consumed capacity of allocating devices
+- can round up and compute user-requesting minimum capacity according to sharing policy range and chunk size
+- can check sharing policy violation
+
+**Distinct Attribute**
+- can prevent allocating the same device in the same request with a distinct constraint
+- can allocate different device in the same request with a distinct contraint
+
+###### Coverage
+
+- `k8s.io/dynamic-resource-allocation/structured:`:	`6/4/2025` - `87.5`
+- `k8s.io/kubernetes/pkg/apis/resource/validation`: `6/4/2025` - `97.8`
+- `k8s.io/kubernetes/pkg/registry/resource/resourceclaim`: `6/5/2025` - `85.5` 
+- `k8s.io/kubernetes/pkg/registry/resource/resourceslice`: `6/5/2025` - `69.1`
+
+##### Integration tests
+
+The existing [integration tests for kube-scheduler which measure performance](https://github.com/kubernetes/kubernetes/tree/master/test/integration/scheduler_perf#readme) will be extended to cover the overheaad of running the additional logic to support the features in this KEP. 
```

Comment:
You can add:

"We will extend the [test for creating large ResourceSlices](https://github.com/kubernetes/kubernetes/blob/78edda21a307ee193fb2ab52210638520af0a1b1/test/integration/dra/dra_test.go#L871-L892) to ensure that a ResourceSlice using the new fields satisfies the etcd limits."